pages/linux/objcopy.md

```diff
@@ -0,0 +1,24 @@
+# objcopy
+
+> Copy the contents of an object file to another.
+> More information: <https://manned.org/objcopy>.
+
+- Copy all data from a specific file to another one:
+
+`objcopy {{path/to/file/source_file}} {{path/to/file/target_file}}`
```

Comment:
```suggestion
`objcopy {{path/to/source_file}} {{path/to/target_file}}`
```
Remember the placeholders need to be concise, so we avoid unnecessary words (such as "your" or "file/target_file", where we have file 2 times). In the case of path placeholders, remember the template:

> {{path/to/file_description}}

Also, we always use "_" to separate each word of the "file_description"

An example of page that follows well this pattern is [feh](https://github.com/tldr-pages/pages/common/feh), see its start:
```md
# feh

> Lightweight image viewing utility.
> More information: <https://feh.finalrewind.org>.

- View images locally or using a URL:

`feh {{path/to/images}}`

- View images recursively:

`feh --recursive {{path/to/images}}`
```